src/main/resources/jail.sh

```diff
@@ -7,44 +7,44 @@ ANALYSIS_FILE=$2 #time.R
 
 DIST_ARCHIVE=$3
 
-sudo tar xzf $DIST_ARCHIVE -C $JAIL
+tar xzf $DIST_ARCHIVE -C $JAIL
```

Comment:
Why `sudo` was removed? Does it mean the application always should be running as root?

Reply:
Good question @wivern, I see a lot of SUDOs in this file for each step which IMHO unnecessary.
$JAIL folder was created by the java process so I think that shell script executed by java should be able to write to it without sudo.
I am not sure about all the use cases, do you have any objections/suggestions?

Reply:
@acumarav Could you check how jail works without `sudo` when EE is running under non-root user?

Reply:
It is from my local machine:
![image](https://user-images.githubusercontent.com/7705583/81163391-f6fb1700-8f8e-11ea-83c9-74538d502ded.png)